easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,64 @@
+easyblock = 'PythonBundle'
+
+name = 'aiohttp'
+version = '3.8.1'
+
+homepage = 'https://github.com/aio-libs/aiohttp'
+description = "Asynchronous HTTP client/server framework for asyncio and Python."
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('typing-extensions', '3.10.0.0'),
```

Comment:
Just to confirm, this is a required dep? (Perhaps due to the additonal extension `charset-normalizer`)

Reply:
It's actually a direct dependency for aiohttp. See https://github.com/aio-libs/aiohttp/blob/1dbfbb433f3f7cd05ca97288805dec9e296d75de/setup.cfg#L55

Reply:
In the easyconfig for 2021b, this one moved into the main Python ec

Reply:
Great, thanks!